packages/core/src/managed/graphql/schema.graphql

```diff
@@ -108,16 +113,16 @@ enum MdLifecycleEventScope {
 }
 
 enum MdLifecycleEventType {
-  BAKE,
+  BAKE
   BUILD
 }
 
 enum MdLifecycleEventStatus {
-  NOT_STARTED,
-  RUNNING,
-  SUCCEEDED,
-  FAILED,
-  ABORTED,
+  NOT_STARTED
```

Comment:
nit: It's kind of annoying that every time you push changes to the schema it gets reformatted and looks different from what's in Keel...

Reply:
Fixed it